http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/NettyConfiguration.java

```diff
@@ -32,6 +32,7 @@
 @SdkInternalApi
 public final class NettyConfiguration {
 
+    public static final int CHANNEL_POOL_CLOSE_TIMEOUT_SECONDS = 1;
```

Comment:
Is this long enough for really big channel pools?

Reply:
I suppose it should be really quick to close all channel pools, but I can increase it to 5 seconds.